gnucash/report/report-core.scm

```diff
@@ -148,6 +148,7 @@
 
 ;; Define those strings here to make changes easier and avoid typos.
 (define gnc:menuname-reports "Reports/StandardReports")
+;; Translators: In reality it is "_Activa & Passiva". The English use sometimes strange terms.
```

Comment:
Activa & Passiva is pretty much everything including income and expense, so is that really the right thing to call the menu? Not that Assets & Liabilities is much better, so maybe we should rename it in 5.0. It has the balance sheets, asset and liability balance pie charts, asset, liability, and net worth balance-over-time charts, the portfolio reports, and the ledger and journal reports. Maybe Balances & Investments would be a better title?

Reply:
Another approach: `Balance oriented` or based vs. `Income oriented` ,but then we should perhaps change the source strings.

Reply:
Yes, I do mean to change the English name (or names) of the menu items. Rearranging the reports in each is also possible, as is adding one or more submenus.

The `Income & Expense` submenu contains various income and expense category charts, the cash flow report and barchart, Income Statement and Profit & Loss report (which I think are the same, will check later), and the Trial Balance report.

Our Cash Flow report is quite different from what [accountants call a cash flow statement](https://www.investopedia.com/terms/c/cashflowstatement.asp) though I suppose one might be able to get there with careful account selection. There's even an [old bug](https://bugs.gnucash.org/show_bug.cgi?id=522171) on the nomenclature. Regardless, whether it's an income/expense report depends on what accounts you select. Trial Balance is also only peripherally an income/expense report in that its primary use is to show whether or not you recorded all of your trading gains and losses--but it does that by examining balances of all accounts in the book.

Balance-oriented and Income-oriented are descriptive but seem to me a bit stilted. I like the idea but I want the wording to percolate for a bit to see if something more fluent comes to mind.